wcfsetup/install/files/acp/templates/systemCheck.tpl

```diff
@@ -129,6 +129,18 @@
 			<small>{lang}wcf.acp.systemCheck.mysql.innodb.description{/lang}</small>
 		</dd>
 	</dl>
+	
+	<dl{if !$results[mysql][foreignKeys]} class="formError"{/if}>
+		<dt>{lang}wcf.acp.systemCheck.mysql.foreignKeys{/lang}</dt>
+		<dd>
+			{if $results[mysql][foreignKeys]}
+				{@$statusOk} {lang}wcf.acp.systemCheck.pass{/lang}
+			{else}
+				{@$statusInsufficient} {lang}wcf.acp.systemCheck.notSupported{/lang}
```

Comment:
Consider giving a more helpful message here: It's not that foreign keys are _unsupported_. The issue is that they (somehow) went missing.